pkg/apis/pipeline/v1alpha1/run_types.go

```diff
@@ -90,6 +94,14 @@ const (
 	RunSpecStatusCancelled RunSpecStatus = "RunCancelled"
 )
 
+// RunSpecStatusMessage defines human readable status messages for the TaskRun.
+type RunSpecStatusMessage string
+
+const (
+	// Run cancelled as the Pipeline running it was cancelled.
```

Comment:
Please give this a valid godoc comment - i.e., one starting with `// RunCancelledByPipelineMsg ...`

Reply:
Done.